### ISSUE TITLE
Google issue

### ISSUE THREAD

**Issue by borut-t** [netbe/Babelish#147]
`Access blocked: This app’s request is invalid`

This is an error when trying to authenticate with google to give access to the spreadsheet.

https://developers.google.com/identity/protocols/oauth2/resources/oob-migration

**Comment by netbe** [comment]
Yeah looking at the google_drive gem I am not sure they have a quick fix for this.

I did found a post by someone recommending a way to get the authorization code.

https://stackoverflow.com/questions/71318804/google-oauth-2-0-failing-with-error-400-invalid-request-for-some-client-id-but/71491500#71491500

**Comment by borut-t** [comment]
> Yeah looking at the google_drive gem I am not sure they have a quick fix for this.
> 
> I did found a post by someone recommending a way to get the authorization code.
> 
> https://stackoverflow.com/questions/71318804/google-oauth-2-0-failing-with-error-400-invalid-request-for-some-client-id-but/71491500#71491500

Any news on this end?

**Comment by rossrossp** [comment]
We are also facing this issue. Has anyone made any progress?

**Comment by DerHlopec** [comment]
Hey, also facing this issue with auth code. Any updates here?

**Comment by borut-t** [comment]
@netbe I think this is an issue we cannot go along with. Either we make a fix or completely shut down this lib since it's useless atm.
